torch/onnx/symbolic_helper.py

```diff
@@ -939,6 +939,17 @@ def quantize_helper(g, tensor, scale, zero_point):
     output = g.op("QuantizeLinear", tensor, scale, zero_point)
     return g.op("prim::TupleConstruct", output, scale, zero_point)
 
+# NOTE: Original `bias` is float in PyTorch. Quantization is applied in kernel.
+#       To mimic behavior in ONNX, perform the `bias` quantization step here.
+#       Then append the dequantization step to ready `bias` for unquantized operators.
+def requantize_bias_helper(g, bias, input_scale, weight_scale):
```

Comment:
A bit confused by the name here. I see we are calculating the quantized bias value and dequantize bias. Did you mean dequantize_bias_helper?

Reply:
indeed, separating the latter dequantize part out.